### DISCUSSION TITLE
Get list of active requests

### DISCUSSION THREAD

**Discussion by lufton** [getgauge/taiko#2621] [General]
Hello. My problem is that I'm trying to run taiko in headless mode and actions like goto/click fails with timeout even though according to screenshots it makes at failure the page is completely loaded. So I wonder if there are some resource loading or request going on that makes taiko think that page is not fully navigated.
Is there a way to check witch resources are currently blocking page to be considered as fully navigated?

**Comment by zabil** [comment]
Don't think this can be done in Taiko. You could try using the [cri](https://docs.taiko.dev/api/client/) to add some additional logs or inspect requests.
